tests/test_utils.py

```diff
@@ -239,3 +239,47 @@ def test_get_instance_token_file_name():
     expected = os.path.join(os.path.expanduser('~'), '.osbs', 'spam.token')
 
     assert get_instance_token_file_name('spam') == expected
+
+@pytest.mark.parametrize(('labels', 'fnc', 'expect'), [
+        ({},
+         ("get_label_only", Labels.LABEL_TYPE_COMPONENT),
+         "com.redhat.component"),
+        ({},
+         ("get_label_only", "doesnt_exist"),
+         KeyError),
```

Comment:
Maybe `Exception`? We're outside the specification of how this function ought to behave, so if it were to raise e.g. `TypeError` that would be perfectly valid behaviour and the test should still pass.